pom.xml

```diff
@@ -101,5 +101,10 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>3.8.1</version>

```

Comment:
Needs to be shaded in. Underneath the version, include the following:
`
<scope>compile</scope>
`

Then add in a relocation for the library by adding:
```
<relocation>
    <pattern>com.google</pattern>
    <shadedPattern>me.gong.mcleaks.util.google</shadedPattern>
</relocation>
```
Under the relocations for the shade plugin

Reply:
I'm guessing you want me to change the pattern directory. I'll change it and you can tell me if I did the right thing